plugins/generator-1.16.5/forge-1.16.5/mappings/villagerprofessions.yaml

```diff
@@ -0,0 +1,15 @@
+_default: VillagerProfession.ARMORER
+_mcreator_prefix: "CUSTOM:"
```

Comment:
Right now, one can't add proffesions so CUSTOM: here is not needed and may be confusing for new users checking the code

Reply:
Ok I removed it and I'll add it when I'll make a PR to add custom Villager Profession mod element